### ISSUE TITLE
Docker volumes prevent updated file replacement on mount

### ISSUE THREAD

**Issue by sarpik** [sarpik/turbo-schedule#9]
TL;DR:

New files from the docker image will NOT take effect
if they already exist in the mounted docker volume,

making it extremely annoying to update some of them, such as:

* **the `latest` symlink** (very important!), pointing to the latest database
* the API docs 
* etc

all are very important,
and currently the fix is to manually update them yourself after deployment --
that sucks!

---

We have 2 docker volume mounts:

* `turbo-schedule--database`  -> `/usr/src/app/server/dist/database`
* `turbo-schedule--generated` -> `/usr/src/app/server/dist/generated`

I would prefer a single one, but there's an issue:

once mounting, assuming you provide the mount points to docker (as you should),
the volume folders would not be replaced (good),
but they would **not** be merged either -- it's either everything or nothing at all.

I've researched this a bit, mostly about docker, and it does not look like there's a solution available,

except from invoking a script once the container is mounted
that would do all the manual work,
which still sucks - you have to launch it manually, it's unstable,
there might be some (or lots!) of downtime in the time frame
from deployment to resolving the issues.

**Comment by sarpik** [comment]
the `openAPI.json` file also cannot get overwritten, meaning that it always stays the same unless we manually generate it locally & copy it over to the docker volume. this sucks.
